### ISSUE TITLE
iOS build seems broken

### ISSUE THREAD

**Issue by Ottunger** [capacitor-community/android-security-provider#1]
**Describe the bug**
When package is installed, `npx cap add|sync ios` will fail

**To Reproduce**
Install package in a demo repo, add ios capacity.

**Expected behavior**
iOS platform would be ready.

**Smartphone (please complete the following information):**
 - Device: -
 - OS: iOS

**Additional context**
Command cap add fails as such:
```
✖ Updating iOS native dependencies with pod install - failed!
✖ update ios - failed!
[error] Analyzing dependencies
        [!] No podspec found for `CapacitorCommunitySecurityProvider` in
        `../../node_modules/@capacitor-community/security-provider`
```

**Comment by Ottunger** [comment]
Renaming file to `CapacitorCommunitySecurityProvider.podspec` and changing name within works.

**Comment by dtarnawsky** [comment]
This issue should be fixed in 5.0.1
